assets/js/addListing.js

```diff
@@ -2,7 +2,12 @@ const submitButton = document.querySelector('#submit');
 let order=1;
 let curentItemId ;
 let nextItemId;
-
+var styling=class{
```

Comment:
O să-ncerc să fiu mai clar: 
* o clasă în css care are `transform: translateY()` și `opacity: 0`
* în script să o folosești în felul `element.classList.toggle('clasa-de-mai-sus')`
* poți schimba din `toggle()` în `add()`  dacă nu plănuiești să faci funcționalitate de `back` în form

Reply:
Nu prea functioneaza asa. Gen mie imi adauga css practic chestia aia. Ideea e ca nu imi rescrie css-ul pe care il am. Si isi salveaza proprietatile initiale.  
Cred ca pot sa gasesc o metoda sa fac toggle sa mearga in situatia asta dar mai tare ma complic. 
![image](https://user-images.githubusercontent.com/22691630/47787412-85963480-dd17-11e8-8c2b-640272aaa6de.png)

Reply:
Pentru ca proprietatile alea deja exista. Eu le rescriu practic, nu le adaug. Either way, accepta pull requestu, sa ma apuc de altceva.